### ISSUE TITLE
Gradle Support?

### ISSUE THREAD

**Issue by wdschei** [JamaSoftwareEngineering/linkchecker-maven-plugin#1]
Have you thought about abstracting this to also create/publish a Gradle plugin?

If not, mind if we do?

**Comment by sverhagen** [comment]
Our build chain heavily leans on Maven, and there's no Gradle in it, so we have little investment, and more importantly, little know-how to do a Gradle plugin, but you are very welcome to use the code in this repository as a basis, to fork it, or maybe even submit a pull request; we Apache-licensed it for that precise reason.

**Comment by wdschei** [comment]
@sverhagen It took this a while to bubble up on our backlog, but we did finally get to doing it. We are already starting to dog food our new version and will be publishing it shortly. Since this was such an invasive set of changes, we went ahead and started from scratch and copy/pasted the actual link checking mechanism. We could however look at making a PR here that would extract that piece out and the `gradle-linkchecker-plugin` could simply depend on the `linkchecker-maven-plugin`. There wouldn't be to much extra baggage, so this would definitely be possible. Here is the [`gradle-linkchecker-plugin`](https://github.com/rackerlabs/gradle-linkchecker-plugin) project.
